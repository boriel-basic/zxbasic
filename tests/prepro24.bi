' A typical inline function

#define REPEAT(X, Y) FOR xxx = 1 TO X \
        Y \
    NEXT 


REPEAT(10, PRINT xxx)

