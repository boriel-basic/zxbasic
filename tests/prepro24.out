




FOR xxx = 1 TO 10 
        PRINT xxx 
    NEXT 
#line 9
PRINT "Line 9"
